app/design/frontend/base/default/template/ebanx/form/card/instalments.phtml

```diff
@@ -33,11 +33,16 @@ $_selected = $this->getInfoData('instalments');
 
         var updateInstalment = function (){
           var text = document.getElementById('cc-<?= $lowerCountry ?>-local-amount');
-          text.innerHTML = '<strong>' + selectInstalment.options[selectInstalment.selectedIndex].getAttribute('data-local-amount') + '</strong>';
+          var instalmentOption=  selectInstalment.options[selectInstalment.selectedIndex];
+          var localAmount = instalmentOption && instalmentOption.getAttribute ? instalmentOption.getAttribute('data-local-amount') : undefined;
```

Comment:
Instead of return undefined to localAmount return false or null, is the best practices :)